src/main/java/de/teamlapen/vampirism/player/vampire/VampirePlayer.java

```diff
@@ -216,15 +245,59 @@ public void biteEntity(int entityId) {
             return;
         }
         if (e != null && e instanceof EntityLivingBase) {
-
             if (e.getDistance(player) <= player.getEntityAttribute(EntityPlayer.REACH_DISTANCE).getAttributeValue() + 1) {
-                biteEntity((EntityLivingBase) e);
+                if (determineBiteType(((EntityLivingBase) e)) == BITE_TYPE.ATTACK) {
+                    biteEntity(((EntityLivingBase) e));
+                } else {
+                    if (victim == -1) biteTickCounter = 0;
+
+                    victim = e.getEntityId();
+                    PotionEffect effect = new PotionEffect(PotionBloodLoss.POTION, 20, 7, false, false);
+                    ((EntityLivingBase) e).addPotionEffect(effect);
+
+                    PotionEffect feedingEffect = new PotionEffect(PotionFeeding.POTION, 25, 1, false, false);
+                    player.addPotionEffect(feedingEffect);
+
+                    NBTTagCompound nbt = new NBTTagCompound();
+                    nbt.setInteger(KEY_VICTIM_ID, victim);
+                    sync(nbt, true);
+
+                }
             } else {
                 VampirismMod.log.w(TAG, "Entity sent by client is not in reach " + entityId);
             }
         }
     }
 
+    public void biteVictim() {
+        EntityLivingBase e = ((EntityLivingBase) player.world.getEntityByID(victim));
+        if (e == null || e.getHealth() == 0f) {
+            endBiting();
+            return;
+        }
+        PotionEffect effect = new PotionEffect(PotionBloodLoss.POTION, 20, 7, false, false);
+        ((EntityLivingBase) e).addPotionEffect(effect);
+
+        PotionEffect feedingEffect = new PotionEffect(PotionFeeding.POTION, 25, 1, false, false);
+        player.addPotionEffect(feedingEffect);
+
+        VampLib.proxy.getParticleHandler().spawnParticles(player.world, ModParticles.FLYING_BLOOD_ENTITY, e.posX + 0.5, e.posY + 0.5, e.posZ + 0.5, 10, 0.1F, player.getRNG(), player, true);
+
+        biteEntity(e);
+        if (!(e.getDistance(player) <= player.getEntityAttribute(EntityPlayer.REACH_DISTANCE).getAttributeValue() + 1) || e.isDead)
```

Comment:
Did some testing and turns out my last suggestion did not help the "delayed feeding stop", we also need to check the health here.